src/Consolonia.Themes/Templates/Controls/CheckBox.axaml

```diff
@@ -21,21 +22,18 @@
                       Background="{TemplateBinding Background}">
                     <Grid Name="border"
                           ColumnDefinitions="Auto,Auto,Auto">
-                        <helpers:SymbolsControl Text="["
-                                                Foreground="{TemplateBinding BorderBrush}" />
-                        <helpers:CaretControl Grid.Column="1"
+                        <console:CaretControl Grid.Column="1"
                                               IsCaretShown="{TemplateBinding IsFocused}">
                             <Panel>
-                                <helpers:SymbolsControl Text=" " />
-                                <helpers:SymbolsControl Name="checkMark"
-                                                        Text="V" />
-                                <helpers:SymbolsControl Name="indeterminateMark"
-                                                        Text="■" />
+                                <TextBlock Text="{StaticResource ThemeCheckBoxNotSelectedSymbol}" />
```

Comment:
If terminal does not support thin caret it's gonna look weird. I'm thinking may be lets switch off  the caret for all themes other than TurboVision? Then we could leave here the old layout

Reply:
I was able to make the caret location just like today by applying a style to offset the caret when using multi character. 
```xaml
   <Style Selector="CheckBox console|CaretControl Rectangle#PART_CaretLine">
       <Setter Property="Margin" Value="1 0" />
   </Style>

   <Style Selector="RadioButton console|CaretControl Rectangle#PART_CaretLine">
       <Setter Property="Margin" Value="1 0" />
   </Style>
```